feat: get nyce page

ddivyansh18 opened the pull request:
- Added instructions on how to import and use Nyce-UI in a project

ddivyansh18 committed:
feat: get nyce page

- Added instructions on how to import and use Nyce-UI in a project

ddivyansh18 committed:
fix: default card shadow

- Fixed the issue of card shadow not displaying in cards page

Aman-Jain1996 reviewed (approved):
Suggestion - Make header nav height greater than the button present there and if possible reduce the nav bar button size 
 -  Add some margin-bottom to sidebar nav so that all elements are properly visible
 - Your main content on every page seems to be left aligned towards sidebar menu , if possible 
    center it
    
- Just a suggestion , try to reduce  size of hero image on homepage so that , buttons are visible 
  in the first screen itself

hrishi82 reviewed (approved):
My thoughts are similar to what aman mentioned above. Rest everything looks good!
Loved the background of the landing page!

ddivyansh18 committed:
fix: navbar width, centered description

- Increased width of navigation bar to accommodate "get docs" button
- Centered component description for all the components

ddivyansh18 commented:
@hrishi82 @Aman-Jain1996 I accommodated the suggestions and have made the required changes, thanks for the feedback
